common/modules/olm/manifests/amq-streams.v1.5.0.clusterserviceversion.yaml

```diff
@@ -331,9 +331,9 @@ spec:
 
     ### What is new in AMQ Streams 1.5
 
-    * Suppor for Apache Kafka 2.5.0 and 2.4.1
+    * Suppor for Apache Kafka 2.5.0 and 2.4.0
```

Comment:
I would remove the 2.4.0 completely - new is only 2.5.0. In Strimzi both 2.4.1 and 2.5.0 were new in 0.18.0.

Reply:
damn you are right, it's about what is new!